tools/minify.ts

```diff
@@ -19,49 +19,70 @@
 import {promises as fsp} from 'fs';
 import * as path from 'path';
 import * as uglify from 'uglify-js';
+import * as yargs from 'yargs';
 
-async function minifyFile(filename: string) {
+interface CliArgs {
+  directory?: string;
+  minifyJs?: boolean;
+}
+async function minifyFile(filename: string, isJs: boolean) {
   const content = (await fsp.readFile(filename)).toString();
-  const output = uglify.minify(content, {
-    expression: true,
-    compress: false, // we need to keep it valid JSON
-    output: {quote_keys: true},
-  });
+  let output;
+  if (!isJs) {
```

Comment:
style nit: it will be a little bit more readable if we do

```ts
let options;
if (isJs) {
  options = { ... };
} else {
  options = { ... };
}
const output = uglify.minify(content, options);
```

so we call `minify` just once in the code.